docs/linking/4-grant-consent.puml

```diff
@@ -18,13 +18,12 @@ autonumber 1 "<b>GRANT-#</b>"
 
 DFSP -> Switch ++: ""POST /consents""\n\
 ""  FSIOP-Source: dfspa""\n\
-""  FSIOP-Destination: central-auth""\n\
 ""  FSIOP-Destination: pispa""\n\
 ""  {""\n\
     ""  consentId: "123",""\n\
     ""  consentRequestId: "456",""\n\
     ""  scopes: [ ""\n\
-    ""    { **accountId: "dfsp.username.1234", // If web, these may change!**""\n\
+    ""    { accountId: "dfsp.username.1234",""\n\
```

Comment:
shouldn't be here `accountNickname` convention used?

Reply:
good point, although I don't think we need it since, the nickname is just displayed to the user up front